pkg/api/convertertoapi_test.go

```diff
@@ -918,9 +920,9 @@ func TestConvertVLabsWindowsProfile(t *testing.T) {
 	}
 
 	for _, c := range cases {
+		c := c
```

Comment:
what's this?

Reply:
This must be set to capture range variable. Otherwise, only one case will be run multiple times. This will cause code coverage drop.
Reference: https://golang.org/pkg/testing/#hdr-Subtests_and_Sub_benchmarks